ValidationLibrary.AzureFunctions/Startup.cs

```diff
@@ -48,6 +48,19 @@ public void Configure(IWebJobsBuilder builder)
                 .AddEnvironmentVariables()
                 .Build();
 
+            // Get all rule classes.
+            var assembly = Assembly.Load("ValidationLibrary.Rules, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null");
+            var validationRules = assembly.GetExportedTypes().Where(c => c.GetInterfaces().Contains(typeof(IValidationRule)));
+
+            // Remove those rules defined by the configuration and the environment variables which should be disabled.
+            validationRules = validationRules.Where(r => config.GetValue<string>("Rules:" + r.Name) != "disable");
```

Comment:
1. Consider string interpolation here
2. Use comparison that ignores case

Reply:
Done.